webpack.config.js

```diff
@@ -18,15 +18,28 @@ module.exports = {
     contentBase: dist,
   },
   plugins: [
-    new CopyPlugin([
-      path.resolve(__dirname, "static")
-    ]),
+    new CopyPlugin({
+      patterns: [
+        path.resolve(__dirname, "static")
+      ]
+    }),
 
     new WasmPackPlugin({
       crateDirectory: __dirname,
       //forceMode: "development",
       watchDirectories: [
       ],
     }),
-  ]
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.wasm$/,
+        type: 'webassembly/sync',
+      }
+    ]
+  },
+  experiments: {
+    syncWebAssembly: true
```

Comment:
Is this why you updated webpack? What does this do?

Reply:
Yeah, to update webpack, I needed to add this. I dunno what it does but I just want some hot reloading.